src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -206,7 +210,12 @@ function MoneyRequestPreviewContent({
                         !onPreviewPressed ? [styles.moneyRequestPreviewBox, containerStyles] : {},
                     ]}
                 >
-                    {hasReceipt && (
+                    {showMapAsImage && (
+                        <View style={styles.reportActionItemImages}>
+                            <ConfirmedRoute transaction={transaction} />
```

Comment:
Coming from https://github.com/Expensify/App/issues/40318, we should've prevented interactivity here to avoid the map being movable whilst still loading.